### DISCUSSION TITLE
Automatic dark mode (aka nightwind), but then as official plugin included please

### DISCUSSION THREAD

**Discussion by DHFW** [tailwindlabs/tailwindcss#8369] [Ideas]
We use [Nightwind](https://github.com/jjranalli/nightwind/issues/54) in our project. And is it great! 
However, not so well supported and up to date with Tailwind 3. 

It would be really great as this could come as an official plugin. Or maybe included "out of the box"!

**Comment by adamwathan** [comment]
Hey! Please keep issues only for bug reports, feature requests belong in discussions as we've set up in our issue templates: https://github.com/tailwindlabs/tailwindcss/issues/new/choose

**Reply by DHFW** [reply]
Whoops. Totally missed the choose option. Else I would have placed it as a discussion here. I know what happened, I clicked the New issue button, but wasn't logged in. After logging in I was redirected back to the "new issue" instead of the choose option list! @adamwathan 

Anyways, what do you think of "automatic" dark mode support?

**Reply by pakholeung37** [reply]
I am looking for the same solutions as Nightwind did!  Writing `dark:` everywhere makes too many boilerplates.

**Comment by colbywhite** [comment]
i'm curious what tailwind's stance on this? should this auto functionality be a part of TW proper? or a community thing?
